services/scan/src/importer.ts

```diff
@@ -236,17 +241,18 @@ export class Importer {
       backInterpretation
     );
 
-    debug('currentPrecinctId=%s', currentPrecinctId);
-    if (currentPrecinctId) {
+    debug('precinctSelection=%o', precinctSelection);
+    if (precinctSelection.kind !== 'AllPrecincts') {
```

Comment:
Isn't this case now impossible?

Reply:
Yes, this is a place where we still have precinct-specific logic in the central scanner. Like with the endpoint, we can remove them.

Reply:
Removed.